index.js

```diff
@@ -31,7 +31,7 @@ const render = async (res, category, slug) => {
 	res.locals.active = category + '/' + slug;
 
 	const cobj = t.find(c => c.slug === category);
-	const dobj = cobj.contents.find(d => d.slug === slug)
+	const dobj = cobj? cobj.contents.find(d => d.slug === slug) : {"name": ""}
```

Comment:
```suggestion
	const dobj = cobj? cobj.contents.find(d => d.slug === slug) : {"name": slug}
```

This matches our logic on line 40